src/app/routes/homePage/getInitialData/index.test.ts

```diff
@@ -0,0 +1,93 @@
+import { Agent } from 'https';
+import KyrgyzHomeFixture from '#data/kyrgyz/homePage/index.json';
+import * as fetchPageData from '../../utils/fetchPageData';
+
+import getInitialData from '.';
+
+process.env.BFF_PATH = 'https://mock-bff-path';
+
+const agent = { cert: 'cert', ca: 'ca', key: 'key' };
+const getAgent = jest.fn(() => Promise.resolve(agent as unknown as Agent));
+
+describe('Home Page - BFF Fetching', () => {
+  afterEach(() => {
```

Comment:
Because we are setting the `process.env.SIMORGH_APP_ENV` in our tests, I think we should try and make sure we clean up afterwards, in case there are other tests which rely on this value.

```suggestion
describe('Home Page - BFF Fetching', () => {
  const originalAppEnv = process.env.SIMORGH_APP_ENV;
  
  afterEach(() => {
    process.env.SIMORGH_APP_ENV = originalAppEnv;
```